Clinical-Genomics/scout
# support for CRAM alignments

This PR will introduce support for visualization of cram alignment files in scout (via igv.js)

**How to test**:
1. I've uploaded CRAM and CRAI files for this case on stage: https://scout-stage.scilifelab.se/cust000/livingox/

![image](https://user-images.githubusercontent.com/28093618/69552805-6233cc80-0f9f-11ea-8649-492666d09f2e.png)


1. The case above, from master branch on stage, should show alignment buttons on the variant pages but the igv.js page shouldn't be able to load any sample track
1. Check out to this branch and check that alignments work for this sample (check alignments from all types of variants)
1. Make sure that samples with BAM files still work (You can use this old case for instance: https://scout-stage.scilifelab.se/cust002/F0027117)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
